specification/ai/Azure.AI.Projects/evaluation-results/models.tsp

```diff
@@ -0,0 +1,73 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "../main.tsp";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Projects;
+
+@doc("Evaluation Result resource Definition")
+@discriminator("EvaluationResultType")
+model EvaluationResult {
+  @doc("Type of Evaluation result")
+  EvaluationResultType: EvaluationResultType;
+
+  @doc("Model Name")
+  ModelName: string;
+
+  @doc("Model Version")
+  ModelVersion: string;
+
+  @doc("Model Asset ID")
+  ModelAssetId: string;
+
+  @doc("Dataset Family")
+  DatasetFamily: string;
+
+  @doc("Dataset Name")
+  DatasetName: string;
+
+  @doc("Metrics")
+  Metrics: Record<string, float64>;
```

Comment:
Top level/schema attributes:
-aggregate metrics
-evaluator schema
-will have multiple models
-benchmarking results: model/data
-BlobUri -> Uri (+Type)

Todo Update: evaluationResult Type initial
-Aggregated Metrics + BlobUri + Properties EvaluationResultType
-BlobUri: points to top-level folder/meta-data file

-follow-up with Asset team